packages/embark-utils/src/web3Utils.ts

```diff
@@ -0,0 +1,51 @@
+import { __ } from "i18n";
+import { balanceRegex } from "./constants";
+const Web3EthAbi = require("web3-eth-abi");
+const Web3 = require("web3");
+
+export function getHexBalanceFromString(balanceString: string, web3: any) {
```

Comment:
Since we require Web3 in this file, and that's the same thing we do in `utils.js`, shouldn't we just remove the need to pass web3 as a parameter?
I think we decided to pass it as a param so as to not require web3, but now we don't care since we already do.